Forbid duplicate setting identifiers. (#2979)
quicwg/base-drafts: draft-ietf-quic-http.md

example, a client might be willing to consume a very large response header,
while servers are more cautious about request size.

> Setting identifiers MUST NOT occur more than once in the SETTINGS frame.  A

Reviewer: ```suggestion
The same setting identifier MUST NOT occur more than once in the SETTINGS frame.  A
```

> The same setting identifier MUST NOT occur more than once in the SETTINGS frame.  A

Reviewer 2: Need to rewrap to fix the build break, I think.

> receiver MAY treat the presence of the same setting identifier more than once as

Reviewer: ```suggestion
receiver MAY treat the presence of duplicate setting identifiers as
```
